### PR TITLE
 Improve Appointment classes

### PR DESCRIPTION
**Pull request by SimWPEric**
Added unit test for Description and DateTime class as well

### CODE REVIEW

**Comment by quzhetao01** [comment]
Good improvements made on the appointment class. LGTM, we just need to remember to get someone to update the appointment test class

File: src/main/java/seedu/address/model/appointment/DateTime.java [hunk 1]
```diff
@@ -1,28 +1,58 @@
...
 
     /**
-     * Retrieves the date and time of the appointment.
-     *
-     * @return The date and time of the appointment.
+     * Returns true if a given string is a valid date time.
      */
-    public LocalDateTime getDateTime() {
-        return dateTime;
+    public static boolean isValidDateTime(String test) {
```
Comment (quzhetao01) [inline src/main/java/seedu/address/model/appointment/DateTime.java:30]: helpful method, good add!

**Review by quzhetao01 — COMMENTED** [review]
Good improvements to the appointment class. LGTM! Just need to remember that we need to get someone to populate the the appointment test cases

**Review by quzhetao01 — APPROVED** [review]
My comments are above, using this review to approve the change

File: src/test/java/seedu/address/model/appointment/DataTimeTest.java
Comment (darrentfy) [inline src/test/java/seedu/address/model/appointment/DataTimeTest.java:1]: LGTM, testing covers both negative and positive test cases

**Review by darrentfy — APPROVED** [review]
LGTM! Test cases are extensive

**Pull request merged**